api/external/infra_proxy/infra_proxy.proto

```diff
@@ -255,12 +255,12 @@ service InfraProxy {
     option (chef.automate.api.iam.policy).resource = "infra:infraServers:{server_id}:orgs:{org_id}:nodes";
     option (chef.automate.api.iam.policy).action = "infra:infraServers:get";
   };
-  rpc DeleteNode (infra_proxy.request.DeleteNode) returns (infra_proxy.response.DeleteNode) {
+  rpc DeleteNode (infra_proxy.request.Node) returns (infra_proxy.response.DeleteNode) {
     option (google.api.http).delete = "/api/v0/infra/servers/{server_id}/orgs/{org_id}/nodes/{name}";
     option (chef.automate.api.iam.policy).resource = "infra:infraServers:{server_id}:orgs:{org_id}:nodes";
     option (chef.automate.api.iam.policy).action = "infra:infraServers:update";
   };
-  rpc UpdateNode (infra_proxy.request.UpdateNode) returns (infra_proxy.response.UpdateNode) {
+  rpc UpdateNode (infra_proxy.request.NodeObject) returns (infra_proxy.response.Node) {
```

Comment:
`UpdateNode` & `CreateNode` now using same request `NodeObject` so we can reuse it.